src/CMakeLists.txt

```diff
@@ -423,11 +446,22 @@ if(NOT APPLE AND NOT FREEBSD)
     PROPERTY LINK_FLAGS "-Wl,-Bsymbolic -Wl,-Bsymbolic-functions")
 endif()
 
-install(
-  TARGETS ceph-common
-  LIBRARY
-  DESTINATION ${CEPH_INSTALL_PKGLIBDIR}
-  NAMELINK_SKIP)
+if(MINGW)
+  install(
+    TARGETS ceph-common
+    RUNTIME
+    DESTINATION ${CEPH_INSTALL_PKGLIBDIR})
+else()
+  install(
+    TARGETS ceph-common
+    LIBRARY
+    DESTINATION ${CEPH_INSTALL_PKGLIBDIR}
+    NAMELINK_SKIP)
+endif()
+
+if(WIN32)
```

Comment:
Yes, I was trying to underline the fact that it's the common lib but we're linking against it only on Windows. A more generic name might be even more confusing.

For what is worth, the dynamic library is safe to use on all platforms, that wouldn't require platform checks. Using the static one would lead to duplicate symbols though.